pkg/dbal/Tests/DbalConnectionFactoryConfigTest.php

```diff
@@ -101,6 +101,72 @@ public static function provideConfigs()
             ],
         ];
 
+        yield [
+            [
+                'dsn' => 'mysql+pdo:',
+                'connection' => [
+                    'dbname' => 'dbname',
```

Comment:
could you please use `customDbName` to reflect that it overwrites default value.

Reply:
Done